Documentation/ApiOverview/SiteHandling/Basics.rst

```diff
@@ -9,17 +9,17 @@ Basics
 ======
 
 .. note::
-   Site handling as described here is available since TYPO3 9 LTS.
+   Site handling is available since TYPO3 9 LTS.
 
-TYPO3 site handling and configuration is the starting point for creating new web sites. The corresponding modules are found in the TYPO3 backend
-in the section "Site management".
+TYPO3 site handling and configuration is the starting point for creating new web sites. 
+The corresponding modules are found in the TYPO3 backend in the section "Site management".
 
 A site configuration consists of the following parts:
 
-* Base URL configuration: Under which domain(s) is my site accessible
```

Comment:
This actually was a question with question mark thus actually OK.

Reply:
The answer stands before. So it cannot be a question any more.